psalm.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0"?>
 <psalm
-    totallyTyped="false"
+    errorLevel="2"
```

Comment:
why level 2?

Reply:
Because its the replacement for `totallyTyped="false"` according to: https://psalm.dev/docs/running_psalm/configuration/#totallytyped

`errorLevel="1"` reported even more errors.

Reply:
authentication doesn't need this, what is the default? I'd prefer to have these common settings the same everywhere.

Reply:
Level 2 is too low, for e.g. the core uses level 4.

Reply:
The default `errorLevel` is 2. We can also drop the `resolveFromConfigFile` definition because we also use the default value.
<img width="1282" alt="image" src="https://user-images.githubusercontent.com/9105243/194747280-402a43a2-ca1c-49ca-8064-7adc6c5fadb7.png">

Reply:
@ADmad Psalm error levels are stricter when going lower. See https://psalm.dev/docs/running_psalm/error_levels/
So level 1 is stricter than level 2.

Reply:
Sorry again :)  I was confused as phpstan's level become stricter with higher number.